python/lsst/sconsUtils/utils.py

```diff
@@ -74,6 +74,23 @@ def libraryPathPassThrough():
 
 
 ##
+#  @brief Returns the full path to the Python executable as determined
+#  from the PATH. Does not return the full path of the Python running
+#  SCons. Caches result and assumes the PATH does not change between
+#  calls.
+pythonPath = None
+
+
+def whichPython():
+    # Call out to "python" and ask it where it is
```

Comment:
I believe that `sconsUtils` is special because `pydoc` won't run on the file and you have to use doxygen. The file can't be imported unless it is being imported by SCons.